includes/class-wc-google-analytics.php

```diff
@@ -536,193 +422,6 @@ private function disable_tracking( $type ) {
-			$this->get_tracking_instance()->add_to_cart( $product );
-			return;
-		}
-
-		// Add single quotes to allow jQuery to be substituted into _trackEvent parameters
-		$parameters             = array();
-		$parameters['category'] = "'" . __( 'Products', 'woocommerce-google-analytics-integration' ) . "'";
-		$parameters['action']   = "'" . __( 'Add to Cart', 'woocommerce-google-analytics-integration' ) . "'";
-		$parameters['label']    = "'" . esc_js( $product->get_sku() ? __( 'ID:', 'woocommerce-google-analytics-integration' ) . ' ' . $product->get_sku() : '#' . $product->get_id() ) . "'";
-
-		if ( ! $this->disable_tracking( $this->ga_enhanced_ecommerce_tracking_enabled ) ) {
-
-			$item = '{';
-
-			if ( $product->is_type( 'variable' ) ) {
-				$item .= "'id': google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ] !== undefined ? google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ].id : false,";
-				$item .= "'variant': google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ] !== undefined ? google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ].variant : false,";
-			} else {
-				$item .= "'id': '" . $this->get_tracking_instance()->get_product_identifier( $product ) . "',";
-			}
-
-			$item .= "'name': '" . esc_js( $product->get_title() ) . "',";
-			$item .= "'category': " . $this->get_tracking_instance()->product_get_category_line( $product );
-			$item .= "'quantity': $( 'input.qty' ).val() ? $( 'input.qty' ).val() : '1'";
-			$item .= '}';
-
-			$parameters['item'] = $item;
-
-			$code                   = $this->get_tracking_instance()->tracker_var() . "( 'ec:addProduct', {$item} );";
-			$parameters['enhanced'] = $code;
-		}
-
-		$this->get_tracking_instance()->event_tracking_code( $parameters, '.single_add_to_cart_button' );
-	}
-
-	/**
-	 * Enhanced Analytics event tracking for removing a product from the cart
-	 */
-	public function remove_from_cart() {
-		if ( ! $this->enhanced_ecommerce_enabled( $this->ga_enhanced_remove_from_cart_enabled ) ) {
-			return;
-		}
-
-		$this->get_tracking_instance()->remove_from_cart();
-	}
-
-	/**
-	 * Adds the product ID and SKU to the remove product link if not present
-	 *
-	 * @param  string $url
-	 * @param  string $key
-	 * @return string
-	 */
-	public function remove_from_cart_attributes( $url, $key ) {
-		if ( strpos( $url, 'data-product_id' ) !== false ) {
-			return $url;
-		}
-
-		if ( ! is_object( WC()->cart ) ) {
-			return $url;
-		}
-
-		$item    = WC()->cart->get_cart_item( $key );
-		$product = $item['data'];
-
-		if ( ! is_object( $product ) ) {
-			return $url;
-		}
-
-		$url = str_replace( 'href=', 'data-product_id="' . esc_attr( $product->get_id() ) . '" data-product_sku="' . esc_attr( $product->get_sku() ) . '" href=', $url );
-		return $url;
-	}
-
-	/**
-	 * Google Analytics event tracking for loop add to cart
-	 */
-	public function loop_add_to_cart() {
```

Comment:
The `loop_add_to_cart` function was only used for Universal Analytics and if gtag was enabled it would [immediately return](https://github.com/woocommerce/woocommerce-google-analytics-integration/blob/39757b6914d7bbeae1d1394e09f32fffd39eaf41/includes/class-wc-google-analytics.php#L619). The add to cart tracking will likely change quite a bit but for now the aim is just to remove all UA code.